src/common/snippets/docs/debug_capabilities/parameters_dump.md

```diff
@@ -0,0 +1,22 @@
+# Snippet parameters dump
+
+Snippet parameters can be captured during the pass flow, which can be useful for debugging and optimization purposes.
+
+To turn on snippet parameters dump feature, the following environment variable should be used:
+```sh
+    OV_SNIPPETS_DUMP_PARAMS=<space_separated_options> binary ...
+```
+
+Examples:
+```sh
+    OV_SNIPPETS_DUMP_PARAMS="snippets=brgemm dir=dump" binary ...
+```
+
+Option names are case insensitive, the following options are supported:
+ - `snippets` : Dump parameters for selected snippets.
```

Comment:
I'd say this debug cap is solely `BrgemmBlocking` specific, so I don't think we need an additional `snippets` param for that: we anyway don't have a necessity of the specific params dumping from other passes.
Let's simplify the logic to make the debug cap setting as simple as possible: the only thing user should specify via env variable is a path to csv file

Reply:
Removed unnecessary details in docs and implementation